depz/depz.py

```diff
@@ -1286,6 +1299,13 @@ def do_fast_forward (self, rname, rr):
     branches = []
 
     current = git.current_branch
+    if current is None:
+      llog.warn("No current branch or in a detached HEAD state. ")
+      if "checkout" in rr._dict:
+        llog.warn("Manually 'git checkout " + rr.val.checkout + "' in the repo.")
+      else:
+        llog.warn("Manually 'git checkout master' (or other default branch) in the repo.")
+      return
```

Comment:
Maybe this should be factored out into a single sanity checking function, since it also appears above, and I think there are other sanity-checks like it already?